src/Ledger/Ratify.lagda

```diff
@@ -263,39 +266,83 @@ actualVotes Γ pparams cc ga votes
   roleVotes r = mapKeys (uncurry credVoter) (filter (λ (x , _) → r ≡ proj₁ x) votes)
 
   activeDReps = dom (filter (λ (_ , e) → currentEpoch ≤ e) dreps)
-  spos = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+  spos  = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+\end{code}
+\begin{code}
 
   getCCHotCred : Credential × Epoch → Maybe Credential
   getCCHotCred (c , e) = case ¿ currentEpoch ≤ e ¿ᵇ , lookupᵐ? ccHotKeys c of
 \end{code}
 \begin{code}[hide]
-    λ where
+      λ where
+\end{code}
+\begin{code}
+        (true , just (just c'))  → just c'
+        _                        → nothing -- expired, no hot key or resigned
+\end{code}
+\begin{code}[hide]
```

Comment:
Okay, I thought we could get away with hiding this because the name is descriptive and since this figure is already running off the page.  I'll unhide `getPoolParams`, and then we can split this figure in two in the final edit.